src/main/java/org/testng/reporters/util/StackTraceTools.java

```diff
@@ -10,25 +10,30 @@
  * @version $Revision: 173 $
  */
 public class StackTraceTools {
```

Comment:
The class should be final

Reply:
fixed. I think this class needs to go away as well (its not being used anywhere in the codebase)

Reply:
Right!